### PR TITLE
Price chart - initial functions

### PR DESCRIPTION
**Pull request by mariarabin**
working code for the different time chart display (code to be enhanced still)

### CODE REVIEW

File: app/routes/htmlRoutes.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// const path = require("path");
```
Comment (alexjzyang) [inline app/routes/htmlRoutes.js]: app folder doesn't exist anymore, Could you please incorporate your changes in the routes folder in the root folder?
Thanks

File: public/js/priceChart.js [hunk 2]
```diff
@@ -0,0 +1,315 @@
+var dataSets = [];
```
Comment (alexjzyang) [inline public/js/priceChart.js:1]: I see you've installed chart.js as a dependancy to the node project. this means we can just require it in the priceChart js file.

File: public/js/priceChart-package.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/*!
```
Comment (alexjzyang) [inline public/js/priceChart-package.js]: We don't need this file anymore because we installed it as a node dependancy

File: public/home.html [hunk 4]
```diff
@@ -1,16 +1,63 @@
...
+            </div>
+        </div>
+    </div>
 </body>
 
-</html>
\ No newline at end of file
+</html>
+
+<script src="/js/priceChart-package.js"></script>
```
Comment (alexjzyang) [inline public/home.html]: Same here, we don't need to include priceChart-package.js because we installed it to Node

**Review by alexjzyang — CHANGES_REQUESTED** [review]

File: public/js/priceChart.js (hunk 2 above)
Comment (mariarabin) [reply public/js/priceChart.js:1]: priceChart.js file si purely all my JS functions - so I did not add it.

**Review by alexjzyang — APPROVED** [review]

**Comment by alexjzyang** [comment]
looking good Maria~

**Pull request merged**